wp-content/plugins/hurumap/db.php

```diff
@@ -100,6 +100,45 @@ function migrate() {
 
     }
 
+    if ($version < 4) {
```

Comment:
Version here is the same as as in https://github.com/TakwimuAfrica/Dashboard/pull/21/files#diff-199986d3b01e1279a89a1dd8f387e586R101. If I've run that migration, my understanding `$version` will now be `4` which means this migration **won't** run.

Reply:
@kilemensi , yes version is the same. Maybe it was what was causing the error on local. But I was hoping this two PRs will be pushed at the same time to production. So no need to add another version

Reply:
Here be dragons!

Reply:
This still bugs me @KhadijaMahanga ... Lets add a story to track it.